scheduler/src/main.rs

```diff
@@ -1,12 +1,23 @@
+use std::env;
+
 use log::{debug, info};
-use scheduler::manager::Manager;
+use scheduler::{config::Config, manager::Manager};
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     env_logger::init();
     info!("starting up");
 
-    let manager = Manager::new();
+    info!("loading config");
+    let mut dir = env::current_exe()?; // get executable path
+    dir.pop(); // remove executable name
```

Comment:
This is `env::current_dir()`.
Also you may want in a future PR to add support for finding the configuration file either through a dedicated environment variable or a CLI option for the scheduler binary.

Reply:
Yes, that would be a nice feature.